src/host/ut_host/ScreenBufferTests.cpp

```diff
@@ -6293,8 +6661,8 @@ void ScreenBufferTests::CursorSaveRestore()
 
     const auto selectAsciiChars = L"\x1b(B";
     const auto selectGraphicsChars = L"\x1b(0";
-    const auto saveCursor = L"\x1b[s";
-    const auto restoreCursor = L"\x1b[u";
+    const auto saveCursor = L"\x1b\x37";
+    const auto restoreCursor = L"\x1b\x38";

```

Comment:
I'm not sure I understand this one

Reply:
This is just swapping the `ANSISYSSC`/`ANSISYSRC` controls for the equivalent `DECSC`/`DECRC` controls. For the most part they're aliases, so it shouldn't matter which of them we use, except that `ANSISYSSC` is disabled when the left-right margin mode is active, because it clashes with the `DECSLRM` sequence. So if we want to test the effect of horizontal margins, we have to use the `DEC` controls.